web/crux/proto/crux.proto

```diff
@@ -940,3 +945,22 @@ message HealthResponse {
   string cruxVersion = 101;
   optional string lastMigration = 102;
 }
+
+// TEMPLATE
+message TemplateResponse {
+  string id = 1;
+
+  string name = 100;
+  string description = 101;
+}
+
+message TemplateListResponse {
+  repeated TemplateResponse data = 1000;
+}
+
+message CreateProductFromTemplateRequest {
+  string id = 1;
+  string accessedBy = 2;
+
+  string productName = 100;
```

Comment:
Currently the template contains the type of product , but I can refactor so that the template contains only images then it'd possible to define what type of product to create.

Reply:
Thanks.